website/package.json

```diff
@@ -10,10 +10,11 @@
     "deploy": "docusaurus deploy"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.62",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.62",
+    "@docusaurus/core": "^2.0.0-alpha.64",
```

Comment:
You also upgraded Docusaurus to the latest release (from today).
this seems unintentional.
Did you verify that the website is functional?

Reply:
Thank you for merging. I found if we upgrade node-fetch to 2.6.1 the docusaurus libs are upgraded to alpha.64 anyway in the yarn.lock, so I put the new version there in packages.json to make it explicit. I did check the website after upgrade, it seems looks fine. But I am not familiar with all the functionalities there. Do you have some specific checks every time you update website?

Reply:
I am just making sure that is builds and comes up. Historically this has been enough to catch any serious problem.